internal/magic/magic.go

```diff
@@ -177,7 +177,9 @@ func newXMLSig(localName, xmlns string) xmlSig {
 // and, optionally, followed by the arguments for the interpreter.
 //
 // Ex:
-//  #! /usr/bin/env php
+//
+//	#! /usr/bin/env php
+//
```

Comment:
Please remove this change. It has nothing to do with the scope of the PR.

Reply:
Hello @gabriel-vasile,
I have made the suggested changes but I am not able to source a JXS image file also I didn't find any converter to convert JPG test file to JXS. Can you help me in getting JXS test image?